tests/testthat/test-reporter-debug.R

```diff
@@ -35,7 +35,7 @@ get_frame_from_debug_reporter <- function(choice, fun, envir = parent.frame()) {
     sink_number = function() 0L,
     with_reporter(
       "debug",
-      test_that("debug_reporter_test", fun())
+      test_that("debug_reporter_test", { fun() })
```

Comment:
an issue with this warning is that `styler` explodes this:

```
styler::style_text('test_that("debug_reporter_test", { fun() })')
# test_that("debug_reporter_test", {
#   fun()
# })
```

It's even worse (IMO) for cases where the test description is long:

```
styler::style_text('
test_that(
  "A really long descriptive name for what this simple test is testing for",
  { expect_true(TRUE) }
)
')
# test_that(
#   "A really long descriptive name for what this simple test is testing for",
#   {
#     expect_true(TRUE)
#   }
# )
```

(this is because of the strict rules around braces in the style guide: https://style.tidyverse.org/syntax.html#indenting)

This is especially cumbersome in cases where we're writing a wrapper to `test_that` like

```
setup_teardown_test_that(x, desc, code) {
  setup(x)
  on.exit(teardown(x))

  test_that(desc, code)
}
```

is requiring `tryCatch()` for the `testthat_braces_warning` (or `suppressWarnings()`) really required here?

We are working on rule that blocks one-expression `{` usage because of all the extra whitespace it causes, it looks like we'll need a carve-out for `test_that()` 😐

Reply:
Can you file an issue and we'll figure out how to weaken this?